airflow/providers/ssh/hooks/ssh.py

```diff
@@ -58,10 +58,13 @@ class SSHHook(BaseHook):  # pylint: disable=too-many-instance-attributes
     :type keepalive_interval: int
     """
 
-    conn_name_attr = 'ssh_conn_id'
-    default_conn_name = 'ssh_default'
-    conn_type = 'ssh'
-    hook_name = 'SSH'
```

Comment:
Somehow I messed up fixing the merge conflict and deleted this earlier - will add back in the morning when I'm at my computer. This should fix the failing CI tests, my bad.